OpenRobertaServer/staticResources/js/helper/util.js

```diff
@@ -324,6 +324,46 @@ define([ 'exports', 'message', 'log', 'jquery', 'jquery-validate', 'bootstrap' ]
     }
     exports.round = round;
 
+    /**
+    * Rounds a number to required decimal and clip for volume
+    *
+    * @param value
+    *            {Number} - to be rounded
+    * @param decimals
+    *            {Number} - number of decimals after rounding
+    * @return {Number} rounded number
+    *
+    */
+    function roundForVolume(value, decimals) {
+        if (value < 0){
+            value = 0;
+        }
+
+        return Number(Math.round(value + 'e' + decimals) + 'e-' + decimals);
+    }
+    exports.roundForVolume = roundForVolume;
+
+    /**
```

Comment:
This should not be here and can probably be solved by rebasing on the latest develop.